3DAmsterdam/Assets/Stadsmodel/Tiles/TileLoader.cs

```diff
@@ -359,6 +366,8 @@ IEnumerator UpdateTerrainTiles(Extent Tempextent)
             {
                 if (TeVerwijderenTiles.ContainsKey(activeTile) == false)
                 {
+                    //Move down tile a notch to avoid Z-fighting in overlapping tiles
+                    activeTiles[activeTile].transform.position = new Vector3(activeTiles[activeTile].transform.position.x, activeTiles[activeTile].transform.position.y-0.01f, activeTiles[activeTile].transform.position.z);
```

Comment:
Je voegt hier 0.01f hardcoded toe, misschien moeten we een offset variable aanmaken hiervoor?

Reply:
Eens! Ga ik doen. Zal sowieso deze file nog even netjes maken, is nog een stuk legacy code, maar die gebruiken we nu nog een tijdje voor de grondtegels totdat de nieuwe PDOK tegels gereed zijn,